v2/tools/generator/internal/astmodel/property_reference_set_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestPropertyReferenceSet_NewPropertyReferenceSet_ReturnsEmptySeT(t *testing.T) {
```

Comment:
minor: Missing t.Parallel() in these tests

Reply:
Added.